webapp/src/app.tsx

```diff
@@ -1349,12 +1349,22 @@ export class ProjectView
     }
 
     pair() {
-        pxt.usb.pairAsync()
-            .then(() => {
-                core.infoNotification(lf("Device paired! Try downloading now."))
-            }, (err: Error) => {
-                core.errorNotification(lf("Failed to pair the device: {0}", err.message))
-            })
+        if (!pxt.commands.webUsbPairDialogAsync) {
```

Comment:
does this mean every target needs to reimplement a pairing dialog? If not present, pair should just call the browser pair dialog directly.